src/components/core/DpEditor/DpEditor.vue

```diff
@@ -313,40 +311,42 @@
 </template>
 
 <script>
-import {
-  Bold,
-  BulletList,
-  HardBreak,
-  Heading,
-  History,
-  Italic,
-  Link,
-  ListItem,
-  OrderedList,
-  Strike,
-  Table,
-  TableCell,
-  TableHeader,
-  TableRow,
-  Underline
-} from 'tiptap-extensions'
 import { CleanHtml, Tooltip } from '../../../directives'
 import {
   Editor, // Wrapper for prosemirror state
   EditorContent, // Renderless content element
-  EditorMenuBar // Renderless menubar
-} from 'tiptap'
-import { createSuggestion } from './libs/editorBuildSuggestion'
+} from '@tiptap/vue-2'
```

Comment:
I actually don't really know what You mean. If You wnt to avoid importing all the single imports in different Places, I think we can leave it like it is. At the moment we only have two Components where tiptap is imported. This one here and DpAnonymizeText.

Reply:
It seemed a bit noisy to me to have so many import statements in a component (since each extension now needs its own), but we can also leave it as it is.

Reply:
I agree that that's too obscure. My idea was to have a separate file, e.g. `extensions.js`, in which we could do the following for each extension:

`export Bold from '@tiptap/extension-bold'`

Then, in DpEditor.vue, we could import the extensions:

`import * as extensions from './extensions.js'`

And then, each extension is available as `extensions.[extensionName]`.
Or we could even import each extension separately:

`import { Bold, BulletList, ... } from './extensions.js'`

And the same for libs.

Reply:
like so?
5637d5b5987e1f6dcf1bcdc8c9da05916d68f102